docs/_guide/02-writing-templates.md

```diff
@@ -286,4 +286,83 @@ html`${cache(data.showDetails
 When lit-html re-renders a template, it only updates the modified portions: it doesn't create or remove any more DOM than it needs to. But when you switch from one template to another, lit-html needs to remove the old DOM and render a new DOM tree. 
 
 The `cache` directive caches the generated DOM for a given binding and input template. In the example above, it would cache the DOM for both the  `summaryView` and `detailView` templates. When you switch from one view to another, lit-html just needs to swap in the cached version of the new view, and update it with the latest data.
+### Rendering nothing
+Very often you may conditionally render something, but in the other case
+you do not want to render anything at all. So, if you are using ternary 
+operators in one case you might return your template and in the other you
+want to return no content lit-html leaves you with a couple options:
+* an empty string (`''`)
+* null
+* undefined
+* nothing sentinel
+#### An empty string
+If you return an empty string somewhere in your template, an empty text node 
+will be rendered, but to the user it will seem like nothing was rendered. 
+
+````js
+${user.isAdmin
+      ? html`<button>DELETE</button>`
+      : ''
+  }
+````
+This does the trick and works great and efficiently in most cases. It does
+not work with slot fallback content.
+#### null or undefined
```

Comment:
I would cut this section entirely. It's not talking about lit, it's talking about DOM APIs. `innerHTML = undefined` might render `undefined`, but ```html`${undefined}` ``` does not.

Reply:
This is again also based on feedback. While it is talking about the way DOM handles this, I think most people might be unaware of this. So, maybe in the intro we should promote the use of empty strings instead of `null` or `undefined`.

Reply:
I added a small disclaimer saying there are inconsistencies. What do you think @arthurevans?